### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by samhamilton**
Hi, 

A small pull to update various deps

Thanks
Sam

### CODE REVIEW

**Comment by beerlington** [comment]
@samhamilton thanks for the PR. It looks like the Ecto bump isn't compatible with Elixir <= 1.6. Any idea if there's a way to specify which version of Ecto to use with each version in Travis CI?

**Comment by samhamilton** [comment]
@beerlington ah no I don't, and as per the [Elixir EOL policy](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/pages/compatibility-and-deprecations.md) 1.6 will only be getting security patches until 1.11 is released so perhaps bump the min Elixir version requirement?

**Comment by psteininger** [comment]
@beerlington and @samhamilton I think we should bump the elixir version. The language is evolving fast and I think most teams are keeping up.

**Comment by psteininger** [comment]
I am slowly returning to a life life other than a F/T dad, as Kindergartens are beginning to open up in Berlin, so I will be doing more maintenance soon :).

**Comment by samhamilton** [comment]
@psteininger I have gone ahead and bumped the package and elixir versions plus also updated travis config for newer versions

**Comment by beerlington** [comment]
Thanks!

File: mix.lock [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+  "makeup_elixir": {:hex, :makeup_elixir, "0.14.0", "cf8b7c66ad1cff4c14679698d532f0b5d45a3968ffbcbfd590339cb57742f1ae", [:mix], [{:makeup, "~> 1.0", [hex: :makeup, repo: "hexpm", optional: false]}], "hexpm", "d4b316c7222a85bbaa2fd7c6e90e37e953257ad196dc229505137c5e505e9eff"},
+  "mime": {:hex, :mime, "1.3.1", "30ce04ab3175b6ad0bdce0035cba77bba68b813d523d1aac73d9781b4d193cf8", [:mix], [], "hexpm", "6cbe761d6a0ca5a31a0931bf4c63204bceb64538e664a8ecf784a9a6f3b875f1"},
+  "nimble_parsec": {:hex, :nimble_parsec, "0.5.3", "def21c10a9ed70ce22754fdeea0810dafd53c2db3219a0cd54cf5526377af1c6", [:mix], [], "hexpm", "589b5af56f4afca65217a1f3eb3fee7e79b09c40c742fddc1c312b3ac0b3399f"},
+  "plug": {:hex, :plug, "1.10.1", "c56a6d9da7042d581159bcbaef873ba9d87f15dce85420b0d287bca19f40f9bd", [:mix], [{:mime, "~> 1.0", [hex: :mime, repo: "hexpm", optional: false]}, {:plug_crypto, "~> 1.1.1 or ~> 1.2", [hex: :plug_crypto, repo: "hexpm", optional: false]}, {:telemetry, "~> 0.4", [hex: :telemetry, repo: "hexpm", optional: true]}], "hexpm", "b5cd52259817eb8a31f2454912ba1cff4990bca7811918878091cb2ab9e52cb8"},
```
Comment (beerlington) [inline mix.lock:18]: I ran my production app's test suite and have a failure due to a breaking change related to this plug update. It has been reported, but I'm going to hold off on a new release until I know a little more - https://github.com/elixir-plug/plug/issues/941

Comment (samhamilton) [reply mix.lock:18]: @beerlington just wanted to know if you are waiting on the next version of plug to be released before you release the ja_serializer version bump?

Comment (beerlington) [reply mix.lock:18]: @samhamilton we're evaluating the plug change on our production app and just need to look into it a little bit. I'm hoping to get this release out very soon though.

Comment (samhamilton) [reply mix.lock:18]: hey @beerlington, how's your evaluation coming? Is there any chance we can get a release cut soonish?

Comment (beerlington) [reply mix.lock:18]: @samhamilton I didn't get a chance to dig in, but I released a new version anyway. I'd rather let the community test and report issues than hold it up any longer. Thanks for the kind nudge!

Comment (samhamilton) [reply mix.lock:18]: cheers!

**Pull request merged**